tools/actions_local_runner.py

```diff
@@ -129,8 +190,7 @@ def main():
     else:
         relevant_steps = grab_all_steps_after(args.all_steps_after, job)
 
-    # pprint.pprint(relevant_steps)
-    asyncio.run(run_steps(relevant_steps, args.job))
+    asyncio.run(run_steps(relevant_steps, args.job, changed_files))  # type: ignore [attr-defined]
```

Comment:
Looks like you have an extra space here:
```suggestion
    asyncio.run(run_steps(relevant_steps, args.job, changed_files))  # type: ignore[attr-defined]
```